### PR TITLE
Allow embedding content in other pages #232

### LINKED ISSUE #232: Allow embedding content in other pages
Currently, embedding content is only allowed in the Schedule page. It's useful to be able to that in other pages as well.

### PR DESCRIPTION
**Pull request by acjh**
Fixes #232 
[Preview](http://rawgit.com/acjh/website/232-embedding-in-other-pages/contents/handbook.html?preview=tutorials)

### CODE REVIEW

**Comment by acjh** [comment]
I'll update [ContentAuthorGuide.md](https://github.com/nus-cs2103/website/blame/9ef9670fb41387d9908fbb07c53277ea2e23ed42/doc/ContentAuthorGuide.md#L37).

**Comment by acjh** [comment]
Updated info in [ContentAuthorGuide.md#embedded-links](https://github.com/acjh/website/blob/232-embedding-in-other-pages/doc/ContentAuthorGuide.md#embedded-links)

**Comment by acjh** [comment]
@damithc Ready for review.

File: contents/handbook.html [hunk 1]
```diff
@@ -11,6 +11,12 @@
 </head>
 <body>
 
+<span class="highlighted">Delay forming teams until your place in a tutorial is confirmed</span>.
+Read the <span
+  onclick="pullContent('handbook-teams', '#embedded-week1-teams', 'Title');"><span
+  class="embedded-link">Teams</span></span> section of the handbook before forming teams.
+<div id="embedded-week1-teams"></div>
+
```
Comment (damithc) [inline contents/handbook.html]: Instead of this, may be use embedding where the handbook refers a FAQ question or an appendix?

**Comment by acjh** [comment]
Updated [preview](http://rawgit.com/acjh/website/232-embedding-in-other-pages/contents/handbook.html?preview=tutorials) for FAQ references in `handbook-tutorials.html`.

**Comment by damithc** [comment]
@acjh Is this something we should worry about?
image

**Comment by acjh** [comment]
Oh, that occurred because I passed a blank `title` to `pullContent(_,_,title,_)`.
We can either use a title, or set `height: 18px;` for `#embedded-heading-container`.

**Comment by damithc** [comment]
> We can either use a title, or set height: 18px; for #embedded-heading-container.

Let's have a fixed height if there is no title, can?

**Comment by acjh** [comment]
*Only* if there is no title? The height with title is `18px`, so there's no conflict.

**Comment by damithc** [comment]
> Only if there is no title? The height with title is 18px, so there's no conflict.

I'm fine as long as we can avoid the situation in screenshot. :-)

**Comment by acjh** [comment]
I've set `min-height: 18px;` instead of `height` in to allow for larger font-size, etc.

**Pull request merged**